supply/lib/supply/options.rb

```diff
@@ -68,12 +68,19 @@ def self.available_options
                                      env_name: "SUPPLY_APK",
                                      description: "Path to the APK file to upload",
                                      short_option: "-b",
+                                     conflicting_options: [:apk_dir],
                                      default_value: Dir["*.apk"].last || Dir[File.join("app", "build", "outputs", "apk", "app-Release.apk")].last,
                                      optional: true,
                                      verify_block: proc do |value|
                                        UI.user_error! "Could not find apk file at path '#{value}'" unless File.exist?(value)
                                        UI.user_error! "apk file is not an apk" unless value.end_with?(value)
                                      end),
+        FastlaneCore::ConfigItem.new(key: :apk_dir,
```

Comment:
`default_value: Dir[File.join("app", "build", "outputs", "apk")].last`
This is the only thing I can think of but it could produce unwanted results if unsigned builds are left in that directory. Is a default required?